w3-account.md

```diff
@@ -12,61 +12,93 @@
+
+Users access their spaces across various user agents operating on multiple devices. Here we introduce [account] primitive designed to enable synchronization of the access across authorized user agents with a user-friendly experience.
 
 ## Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
 
 # Introduction
 
-In w3 family of protocols, a namespace is identified by a [`did:key`] identifier, which means that the owner of that namespace _(private key holder)_ has absolute authority over it. They can delegate some or all of the capabilities for the resources under that namespace to any other [principal]. However, managing these delegations across multiple user agents on multiple devices presents several user experience challenges:
+In W3 protocol, a namespace, or space for short, corresponds to an asymmetric keypair and is identified by a [`did:key`] URI. The private key holder, is the [owner] of that namespace and has absolute authority over it. They can delegate limited or absolute authority over the namespace to any other [principal]. However, managing delegations across multiple user agents on multiple devices presents several user experience challenges:
+
+1. To synchronizing namespace access across user agents they need to discover their [`did:key`] identifiers and level of access granted to them.
+2. Recovering access in case when all devices are lost becomes impossible.
+
+To address these issues, we propose the concept of an account. Account SHOULD have a human-meaningful identifier such as email address. We propose use of email addresses as account identifiers so that derived [`did:mailto`] can act as the [principal] in the [UCAN] delegation chain. This creates [principal] that can be used to aggregate capabilities and manage them.
 
-1. Synchronizing delegations to namespaces across multiple user agents on multiple devices is difficult because of the use of non-memorable [`did:key`] identifiers.
-2. Recovering access if the user loses access to all devices is also a challenge.
+Account can be used to solve both discovery and recovery problems:
 
-To address these issues, we propose the concept of an account as a way to aggregate and manage capabilities under a human-meaningful identifier such as an email address.
+1. Instead of user agents trying discovering each to delegate capabilities with one another, all capabilities get delegated to an account which is then used to re-delegate them as necessary to individual agents.
+2. Recovery is possible even if all devices has been lost as long as the user retains control of their email, because account can always delegate capabilities to new agents.
```

Comment:
```suggestion
2. Recovery is possible even if all devices have been lost as long as the user retains control of their email, because an account can always delegate capabilities to new agents.
```